src/Modules/Build.lua

```diff
@@ -843,11 +843,15 @@ function buildMode:Save(xml)
 	for _, id in ipairs(self.spectreList) do
 		t_insert(xml, { elem = "Spectre", attrib = { id = id } })
 	end
+	local addedStatNames = { SkillDPS = true }
 	for index, statData in ipairs(self.displayStats) do
-		if statData.stat then
-			local statVal = self.calcsTab.mainOutput[statData.stat]
-			if statVal then
-				t_insert(xml, { elem = "PlayerStat", attrib = { stat = statData.stat, value = tostring(statVal) } })
+		if not statData.flag or self.calcsTab.mainEnv.player.mainSkill.skillFlags[statData.flag] then
```

Comment:
What kind of stats does this remove from the output?  Seems like there might be some that are desired from skills other than the main, no?

Reply:
You are right, but that's already the case. For example, `Speed` (aka Attack Speed) changes based on what the main skill is (e.g., Cyclone versus Flame Dash as both become the `Speed` stat; one from Attack Speed and one from Cast Speed). There is no way to guarantee data for "other than the main" are correct without leveraging FullDPS and that's currently up to the user to opt-in to.

Reply:
That's true, even if other tools rely on something like that, it's probably wrong anyway